lib/adapter.js

```diff
@@ -100,9 +100,19 @@ var objectTrackingCallbackInterval =
 // Metrics API Use function
 function use(callback) {
 
+  exports.on('watchdogActivationCount', function(count) {
+    callback('watchdog.activations.count', count);
+  });
+
   // The callback needs to fire every 60 seconds with data from the last 60
   // seconds
   setInterval(function() {
+
+    if (appmetrics && appmetrics.watchdogActivationCount) {
+      exports.emit('watchdogActivationCount',
```

Comment:
I'll have to look closer, but this looks weird.... someone calls .use(cb)... and you use that to decide to start emitting an event? it seems to be intertwining independent notification mechanisms - but I don't recall exactly how this worked in strong-agent, are you trying to duplicate something weird there?